readme.md

```diff
@@ -26,7 +40,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+The jQuery library includes AJAX (Asynchronous JavaScript and XML), which enables asynchronous server side calls to refresh information on the page without a full browser refresh. The $.ajax call requires that you specify a URL, an HTTP method (GET, POST, PUT, etc), and a data-type, and then it responds with a JSON object which you can then make use of in the promises that follow (.done(function(response){}), .fail(), etc).
```

Comment:
:+1: 
I recommend you include that it is sending a request to the url, which responds with a JSON object...